snap/local/runtime-helpers/bin/install-setup-postgres.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+# setup postgres db config file with env vars replaced
+if [ ! -f "$SNAP_DATA/etc/postgresql/10/main/postgresql.conf" ]; then
+    mkdir -p "$SNAP_DATA/etc/postgresql/10/main"
+    cp "$SNAP/etc/postgresql/10/main/postgresql.conf" "$SNAP_DATA/etc/postgresql/10/main/postgresql.conf"
+    # do replacement of the $SNAP, $SNAP_DATA, $SNAP_COMMON environment variables in the config files
+    sed -i -e "s@\$SNAP_COMMON@$SNAP_COMMON@g" \
```

Comment:
When I committed this script for Hanoi, I don't think I actually checked if the `postgresql.conf` file actually contains any `SNAP` vars? We should check and if it doesn't excise this code.

Reply:
it contains a single SNAP_COMMON var:

```
unix_socket_directories = '$SNAP_COMMON/sockets'
```

which gets changed to

```
unix_socket_directories = '/var/snap/edgexfoundry/common/sockets'
```